compiler-rt/lib/scudo/standalone/secondary.h

```diff
@@ -249,11 +273,17 @@ template <typename Config> class MapAllocatorCache {
     }
     do {
       ScopedLock L(Mutex);
+
+      Time = getMonotonicTimeFast();
+      if (Entry.Time != 0)
+        Entry.Time = Time;
+
```

Comment:
I know we will remove the `Entry.Time` but it's still better to have some comment on why we move the `getMonotonicTimeFast()` here.